configuration/amd64/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     tty: true
 
   v2xhub:
-    image: usdotfhwaops/v2xhubamd:carma-streets
+    image: usdotfhwaops/v2xhubamd:latest
```

Comment:
why are we setting this to carma-streets

Reply:
Because it is not the latest that include all the changes like portdrayage, HSM, and sonar scan etc.

Reply:
If you have latest as the image, you will be running a develop image instead of the carma-streets image. It will not have the carma-streets image on it. This change does not make sense to me unless you want to merge this branch into develop right??

Reply:
This should be set to carma-streets not latest right?

Reply:
@paulbourelly999  addressed